utils/zerovec/src/map/map.rs

```diff
@@ -292,6 +292,11 @@ where
             &'b <K as ZeroMapKV<'a>>::GetType,
             &'b <V as ZeroMapKV<'a>>::GetType,
         ),
+    > + ExactSizeIterator<
```

Comment:
Nit: These changes in `map` could stand by themselves. Consider putting them in their own PR.

Reply:
Created PR #2580 for this.

Reply:
Please sync